splitio/api/client.py

```diff
@@ -212,6 +249,7 @@ async def get(self, server, path, apikey, query=None, extra_headers=None):  # py
                 timeout=self._timeout
             ) as response:
                 body = await response.text()
+                await self._record_telemetry(response.status, get_current_epoch_time_ms() - start)
```

Comment:
Is it `response.status` or should be `response.status_code` ? In case of the latter, check other occurrences of `response.status`

Reply:
Yeah, this is a difference between aiohttp (for asyncio) and requests library for threading.